database/seeders/DatabaseSeeder.php

```diff
@@ -13,6 +14,11 @@ class DatabaseSeeder extends Seeder
      */
     public function run()
     {
-        // \App\Models\User::factory(10)->create();
+        $this->call([
+            FacultySeeder::class,
+            DepartmentSeeder::class,
+        ]);
+        
+        User::factory(10)->create();
```

Comment:
You can just write this in UserSeeder then call it just like you did for the others.